docs/sources/panels/configure-data-links/index.md

```diff
@@ -66,9 +66,23 @@ Value-specific variables are available under `__value` namespace:
 
 When linking to another dashboard that uses template variables, select variable values for whoever clicks the link.
 
-`${myvar:queryparam}` - where `myvar` is a name of the template variable that matches one in the current dashboard that you want to use.
+- `${myvar:queryparam}` - where `myvar` is the name of the template variable that matches a variable in the dashboard you want to use.
 
-If you want to add all of the current dashboard's variables to the URL, then use `__all_variables`.
+| State        | Result                                                                        |
+| ------------ | ----------------------------------------------------------------------------- |
+| one selected | `value`                                                                       |
```

Comment:
```suggestion
| `one selected` | `value`                                                                       |
```
Are these states verbatim? If so, use back ticks to denote that they are literal.

Reply:
Hi @osg-grafana, "one selected" and "multiple" (selected) are not verbatim, but "`All`" is indeed. The first two represent the action of selecting one or more values of the template variable. The last one is a choice among the values of a template variable that exists if the variable has "Include All option" set to `true`.
That said, I'm not sure if `value` should be in backticks either. It is not exactly the value, but represents the value whatever it would be. I changed most of it to make it more clear and because I checked and couldn't find use case that covers one of the tables as they were. Now only one table is left and it describes what really happens when you edit data links. What do you think?